test/Data/Graph/WrapperSpec.hs

```diff
@@ -21,3 +21,34 @@ spec = do
       it "returns the empty list" $ do
         reachableVertices (fromList [] :: Graph Integer ()) 0
           `shouldBe` []
+
+  describe "Eq Graph" $ do
+    context "when the graphs are identical" $ do
+      it "returns True" $ do
+        let g1 = fromList [('a', 'a', ['a', 'b', 'c']), ('b', 'b', ['b', 'c']), ('c', 'c', ['a'])]
+        let g2 = fromList [('a', 'a', ['a', 'b', 'c']), ('b', 'b', ['b', 'c']), ('c', 'c', ['a'])]
+        g1 == g2 `shouldBe` True
+
+    context "when the graphs are isomorphic" $ do
+      it "returns False" $ do
+        let g1 = fromList [('1','a',['2']), ('2','b',[])]
+        let g2 = fromList [('2','a',['1']), ('1','b',[])]
```

Comment:
Sorry, could you add in the spaces after the commas again? :)

Reply:
oops! old habits die hard :)

fixed.